ext/dom/document.c

```diff
@@ -1540,7 +1540,7 @@ PHP_METHOD(DOMDocument, save)
 	zval *id;
 	xmlDoc *docp;
 	size_t file_len = 0;
-	int bytes, format, saveempty = 0;
+	int saveempty = 0;
```

Comment:
Aside: Should this also be a `bool`?

Reply:
Hmm, the original value isn't necessarily 0 or 1, but maybe.